portfolio/src/main/webapp/style.css

```diff
@@ -14,101 +16,310 @@
     margin-right: auto;
     width: 50%;
 }
-body {
-    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
-}
-h1 {text-align: center;}
+
+/* New portfolio styling */
+
 *{
-padding:0;
-margin:0;
-text-decoration: none;
-list-style: none;
-box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+	font-family: 'Josefin Sans', sans-serif;
+	box-sizing: border-box;
 }
-
-body{
-    font-family: montserrat;
+.hero{
+	height: 100vh;
+	width: 100%;
+	background-image: url(/images/background.jpg);
+	background-size: cover;
```

Comment:
[2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation): Indent by 2 spaces at a time.

This entire file has indentation issues. You need to configure your **Editor: Tab Size** to be 2, then to fix most of the issues dedent the entire file and then indent once.